cluster/kubemark/gce/config-default.sh

```diff
@@ -19,8 +19,10 @@
 # gce/util.sh script which assumes config filename), but if some things that
 # are enabled by default should not run in hollow clusters, they should be disabled here.
 
-# shellcheck disable=SC2034 # Variables sourced in other scripts.
+# shellcheck disable=SC2034
```

Comment:
Is this what `kube::util::sourced_variable` is also trying to defeat?

Reply:
I'll take a look at this now.